src/manifests_base.py

```diff
@@ -251,3 +248,94 @@ def from_space_separated(cls, tolerations: str) -> List["CephToleration"]:
             return [cls._from_string(toleration) for toleration in tolerations.split()]
         except ValueError as e:
             raise ValueError(f"Invalid tolerations: {e}") from e
+
+
+class ProvisionerAdjustments(Patch):
+    """Update provisioner manifest objects."""
+
+    PROVISIONER_NAME: str
+    PLUGIN_NAME: str
+
+    def tolerations(self) -> Tuple[List[CephToleration], bool]:
+        return [], False
+
+    def adjust_container_specs(self, obj: AnyResource) -> None:
+        csidriver = cast(SafeManifest, self.manifests).csidriver
+        original_dn, updated_dn = csidriver.default_name, csidriver.formatted
+        kubelet_dir = self.manifests.config.get("kubelet_dir", "/var/lib/kubelet")
+
+        for c in obj.spec.template.spec.containers:
+            for idx in range(len(c.args)):
+                if original_dn in c.args[idx] and updated_dn not in c.args[idx]:
+                    c.args[idx] = c.args[idx].replace(original_dn, updated_dn)
+                if "/var/lib/kubelet" in c.args[idx]:
+                    c.args[idx] = c.args[idx].replace("/var/lib/kubelet", kubelet_dir)
+            for m in c.volumeMounts:
+                m.mountPath = m.mountPath.replace("/var/lib/kubelet", kubelet_dir)
+        for v in obj.spec.template.spec.volumes:
+            if v.hostPath:
+                v.hostPath.path = v.hostPath.path.replace("/var/lib/kubelet", kubelet_dir)
+                v.hostPath.path = v.hostPath.path.replace(original_dn, updated_dn)
```

Comment:
Hmm... that repeated `alt.alt.alt.alt.alt.alt.alt.alt.alt.rdb.csi.ceph.com` could end up here too... need to check on that.

Reply:
I checked, and all was ok, I don't see the same name extending